[coc.nvim] The "svelte" server crashed 5 times in the last 3 minutes. The server will not be restarted.

Hi.

After installation, features such as auto-completion do not work.

The output shows “Error: Cannot find module 'typescript'”. `npm install typescript` from `~/.config/coc/extensions/node_modules/coc-svelte` fixed the issue.

> The output shows “Error: Cannot find module 'typescript'”. `npm install typescript` from `~/.config/coc/extensions/node_modules/coc-svelte` fixed the issue.


So it works

Hope it will be fixed soon.

Thanks!

--- a/package.json
+++ b/package.json
@@ -279,7 +279,8 @@
     "dependencies": {
         "lodash": "^4.17.19",
         "svelte-language-server": "^0.10.44",
-        "vscode-languageserver-protocol": "^3.15.3"
+        "vscode-languageserver-protocol": "^3.15.3",
+        "typescript": "^3.9.5"
     },
     "devDependencies": {
         "@tsconfig/node12": "^1.0.0",
@@ -288,7 +289,6 @@
         "coc.nvim": "^0.0.79",
         "js-yaml": "^3.14.0",
         "npm-run-all": "^4.1.5",
-        "rimraf": "^3.0.2",
-        "typescript": "^3.9.5"
+        "rimraf": "^3.0.2"
     }
 }

